addOns/selenium/src/main/java/org/zaproxy/zap/extension/selenium/Browser.java

```diff
@@ -51,7 +51,9 @@ public enum Browser {
      */
     @Deprecated
     INTERNET_EXPLORER("ie", false),
+    @Deprecated(since = "15.12.0", forRemoval = true)
```

Comment:
Needs to be updated to match the latest version, same in the add-on dependencies.